.gitlab-ci.yml

```diff
@@ -948,63 +951,55 @@ workflow:
 .on_testing_cleanup:
   - <<: *if_testing_cleanup
 
+.security_agent_change_paths: &security_agent_change_paths
```

Comment:
If I understand correctly, this block is used both in rules for:
- the security-agent kitchen functional tests
- the security-agent KMT tests

However, I don't know if these two sets of tests should depend on the same exact list of files? For instance, I'd expect the security-agent KMT tests to depend on parts of `test/new-e2e`.

Reply:
I added some paths to this block for consistency with the system-probe agent paths. I think that while we maintain both KMT and kitchen in parallel it's interesting that both run at the same time, so that we can compare results.